tests/unit/utils/data.test.ts

```diff
@@ -311,6 +339,7 @@ describe('createZipFromObject()', () => {
 
   it('serializes binary data as binary file', async () => {
     const zipFile = await createZipFromObject(data);
```

Comment:
as we are testing a bug fix for binary data loaded as ArrayBuffer we may want to ensure the test data are the type we expected (in this case what was declared as ArrayBuffer was not ArrayBuffer but a nodejs Buffer also instanceof Uint8Array)

the suggestion bellow add checks on data type

```suggestion
    // ensure some data are Uint8Array (nodejs)
    expect(
      data.nested.object.with.binary.data.pngImage instanceof ArrayBuffer
    ).toBe(false);
    expect(
      data.nested.object.with.binary.data.pngImage instanceof Uint8Array
    ).toBe(true);
    // ensure some data are ArrayBuffer (web)
    expect(
      data.nested.object.with.binary.data.pdfFile instanceof ArrayBuffer
    ).toBe(true);
    expect(
      data.nested.object.with.binary.data.pdfFile instanceof Uint8Array
    ).toBe(false);
    const zipFile = await createZipFromObject(data);
```